news/2024/2024-10-09-resurrection-cup-2024-results.md

```diff
@@ -0,0 +1,417 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8266808" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-[Boy]DaLat.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/8266808">[Boy]DaLat</a></p>
+
+So I started developing overlays with the aim to provide a more customized, personalized overlay for each tournament. I believe that there are many potentials to how we could improve the default layout.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/12840110" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-Phreel.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/12840110">Phreel</a></p>
+
+What were your biggest fears or concerns while developing the overlay? Was it difficult of a transition?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/7696512" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-Hoaq.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/7696512">Hoaq</a></p>
+
+Totally. Using 3rd-party tool is always like living on the edge. In the past, each times osu! (more specifically the cutting-edge release) gets updated, our custom overlay would stop working, unless the tool we use to run the overlay gets updated as well.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8266808" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-[Boy]DaLat.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/8266808">[Boy]DaLat</a></p>
+
+I would say transitioning from osu!(lazer) to custom overlay might need a little bit of work. Each overlay has its own set of defined workflows, comparing to the standardized flow of osu!(lazer). Streamers, to some extent, need to be introduced to a different workflow, user experience, etc, which might take a while to adapt to. At first, developing overlays are pretty much trivial, you read the state from tosumemory, compare changes, update the state on the UI. But when I started to scale things up, for example, when I have to add more custom state, more interactions, I need to change my way of thinking over and over again. Because things might break miserably if I do not engineering thoroughly. That is why when I was asked to add feature abcdxyz to the overlay, during the production of the tournament, I got really upset because now I had to expand the structure of the overlay in such a short amount of time to keep up with the matches.
```

Comment:
```suggestion
I would say transitioning from osu!(lazer) to a custom overlay might need a little bit of work. Each overlay has its own set of defined workflows, comparing to the standardized flow of osu!(lazer). Streamers, to some extent, need to be introduced to a different workflow, user experience, etc, which might take a while to adapt to.

At first, developing overlays is pretty much trivial: you read the state from tosumemory, compare changes, and update the state on the UI. But when I started to scale things up, for example, when I have to add more custom state or more interactions, I need to change my way of thinking over and over again, because things might break miserably if I do not engineering thoroughly.

That is why when I was asked to add feature abcdxyz to the overlay, during the production of the tournament, I got really upset because now I had to expand the structure of the overlay in such a short amount of time to keep up with the matches.
```